features/step_definitions/admin_auction_form_steps.rb

```diff
@@ -90,7 +90,7 @@
 
 Then(/^I should see the updated delivery deadline$/) do
   within('.estimated-delivery-date') do
-    delivery_date = DefaultDateTime.new(5.business_days.after(DcTimePresenter.convert(3.business_days.from_now))).convert
+    delivery_date = DefaultDateTime.new(5.business_days.after(@end_day)).convert
```

Comment:
it seems like this is reverting the last change you made to address this issue: https://github.com/18F/micropurchase/commit/1a18a17ddf3c7d0acee09937f9ed84d0579ea82f#diff-a20386f4c76aafe8471bb849a3b37f5f

Not opposed to revert, but since this issue popped up before that change was made, seems like this was likely not the culprit